lib/libzfs_core/libzfs_core.c

```diff
@@ -340,6 +344,12 @@ lzc_exists(const char *dataset)
 	return (ioctl(g_fd, ZFS_IOC_OBJSET_STATS, &zc) == 0);
 }
 
+int
+lzc_sync(const char *pool_name)
```

Comment:
Given that this is supposed to be a stable interface, I would like to see a `nvlist_t *opts` variable here that is passed to the kernel and ignored.